src/main/java/frc/robot/auto/generator/PathPlannerAutoGenerator.java

```diff
@@ -50,69 +71,124 @@ public PathPlannerAutoGenerator(DriveSubsystem drive, ArmSubsystem arm, ShooterS
+        //TODO more poses and correct rotation
+    }
+
+    private void setupShuffleboard() {
+        field = new Field2d();
+        tab = Shuffleboard.getTab("Auto");
+        tab.add("Auto Selection", autoSelector).withPosition(0, 0).withSize(2, 1);
+        tab.add("Starting Pose", field).withPosition(0, 1).withSize(6, 4);
+        tab.addString("Alliance", () -> AllianceUtil.isRedAlliance() ? "Red" : "Blue").withPosition(6, 1);
+        delay = tab.add("Auto delay seconds", 0).withSize(2, 1).withPosition(3, 0).getEntry();
+        tab.add("Extra pathfind", extraPathfindPoses).withSize(2, 1).withPosition(5, 0);
+    }
+
     private void registerCommands() {
+        NamedCommands.registerCommand("shootSequence", new ShootSequence(shooter, undertaker, arm, led, autoContext));
+
+        NamedCommands.registerCommand("closeAShot", new ConfigurableShootSequence(shooter, undertaker, arm, led, () -> Constants.Auto.CLOSE_A_SHOT_ANGLE));
+        NamedCommands.registerCommand("closeBShot", new ConfigurableShootSequence(shooter, undertaker, arm, led, () -> Constants.Auto.CLOSE_B_SHOT_ANGLE));
+        NamedCommands.registerCommand("closeCShot", new ConfigurableShootSequence(shooter, undertaker, arm, led, () -> Constants.Auto.CLOSE_C_SHOT_ANGLE));
+        NamedCommands.registerCommand("closerCShot", new ConfigurableShootSequence(shooter, undertaker, arm, led, () -> Constants.Auto.CLOSER_C_SHOT_ANGLE));
+        NamedCommands.registerCommand("behindCloseBShot", new ConfigurableShootSequence(shooter, undertaker, arm, led, () -> Constants.Auto.BEHIND_CLOSE_B_SHOT_ANGLE));
+        NamedCommands.registerCommand("sourceSideShot", new ConfigurableShootSequence(shooter, undertaker, arm, led, () -> Constants.Auto.SOURCE_SIDE_SHOT_ANGLE));
+
         NamedCommands.registerCommand("intake", new AutoIntakeNote(shooter, undertaker)); //blocking
         NamedCommands.registerCommand("intakeUntil", new AutoIntakeNote(shooter, undertaker).withTimeout(0.5)); //blocking
 
         NamedCommands.registerCommand("armHome", new AutoArmHome(arm)); //blocking
-        NamedCommands.registerCommand("armPodiumAngle", new AutoArmMove(arm, Constants.Arm.LONG_SHOT_ANGLE)); //blocking
 
-        NamedCommands.registerCommand("armNoteLeft", new AutoArmMove(arm, Constants.Arm.AUTO_LEFT_NOTE_ANGLE)); //blocking
-        NamedCommands.registerCommand("armNoteMiddle", new AutoArmMove(arm, Constants.Arm.AUTO_MIDDLE_NOTE_ANGLE)); //blocking
-        NamedCommands.registerCommand("armNoteRight", new AutoArmMove(arm, Constants.Arm.AUTO_RIGHT_NOTE_ANGLE)); //blocking
+        NamedCommands.registerCommand("armNoteLeft", new AutoArmMove(arm, Constants.Auto.CLOSE_A_SHOT_ANGLE)); //blocking
+        NamedCommands.registerCommand("armNoteMiddle", new AutoArmMove(arm, Constants.Auto.CLOSE_B_SHOT_ANGLE)); //blocking
+        NamedCommands.registerCommand("armNoteRight", new AutoArmMove(arm, Constants.Auto.CLOSER_C_SHOT_ANGLE)); //blocking
```

Comment:
These can be dropped for now. We might do something to chose faster autos (move arm while driving to next shot spot) but if we do we need to rethink it